### PR TITLE
Add tests for empty args in cli

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: test/lib/cli/health_command_test.rb [hunk 1]
```diff
@@ -78,6 +78,20 @@ module ForemanMaintain
         end
         run_cmd(['--tags=pre-upgrade-check'])
       end
+
+      it 'raises errors on empty arguments' do
+        assert_cmd <<OUTPUT, %w(--label)
+ERROR: option '--label': value not specified
+
```
Comment (Akasurde) [inline test/lib/cli/health_command_test.rb]: LGTM

**Comment by iNecas** [comment]
Thanks @Akasurde for review
